source/dom.js

```diff
@@ -79,11 +81,11 @@ var dom = module.exports = {
      * Find all nodes by selector
      *
      * @param {String} selector
-     * @param {Node} node
+     * @param {Node} [node]
      * @returns {NodeList}
```

Comment:
Yeah, you’re right. (Feel free to open GitHub reviews requesting the changes in such cases, BTW)